index.html

```diff
@@ -57,7 +57,7 @@ <h1>Hey there. I’m Daniel <br> <span class="desc">I’m a software developer</
         </ul>
       </article>
     </section>
-    <button class="down-icon"><img class="down-icon" src="./Images/disabled.svg" alt="" /></button>
+    <!-- <button class="down-icon"><img class="down-icon" src="./Images/disabled.svg" alt="" /></button> -->
```

Comment:
- [ ] to keep your code clean & professional , kindly remove any commented from your repo 🙏